### DISCUSSION TITLE
Retrieve additional context based on chat?

### DISCUSSION THREAD

**Discussion by phillee** [brianpetro/obsidian-smart-connections#599] [General]
If I understand correctly, the Smart Chat feature retrieves notes similar to the current note at the beginning of the chat. 

It seems like chat would be more useful if it retrieves additional notes based on subsequent messages. Or am I not using Smart Chat correctly?

**Comment by christianngarcia** [comment]
The Smart View returns results based on the current note.

However, the Smart Chat is different and returns results based on your conversation 🌼
